node/src/chain_spec/mod.rs

```diff
@@ -42,46 +42,53 @@ use zeitgeist_primitives::{
-            let expected_annual_amount = ztg::COLLATORS * zeitgeist_primitives::constants::BASE;
-            let round_inflation_parts = annual_inflation.deconstruct() as u64 / rounds_per_year;
-            let round_inflation = Perbill::from_parts(round_inflation_parts as _);
-
-            pallet_parachain_staking::InflationInfo {
-                annual: pallet_parachain_staking::Range {
-                    ideal: annual_inflation,
-                    max: annual_inflation,
-                    min: annual_inflation,
-                },
-                expect: pallet_parachain_staking::Range {
-                    ideal: expected_annual_amount,
-                    max: expected_annual_amount,
-                    min: expected_annual_amount,
-                },
-                round: pallet_parachain_staking::Range {
-                    ideal: round_inflation,
-                    min: round_inflation,
-                    max: round_inflation,
-                },
+        macro_rules! generate_inflation_config_function {
+            ($runtime:ident) => {
+                use sp_runtime::Perbill;
+
+                pub(super) fn inflation_config(
+                    annual_inflation_min: Perbill,
+                    annual_inflation_ideal: Perbill,
+                    annual_inflation_max: Perbill,
+                    total_supply: zeitgeist_primitives::types::Balance
+                ) -> pallet_parachain_staking::inflation::InflationInfo<zeitgeist_primitives::types::Balance> {
+                    fn to_round_inflation(annual: pallet_parachain_staking::inflation::Range<Perbill>) -> pallet_parachain_staking::inflation::Range<Perbill> {
+                        use pallet_parachain_staking::inflation::{
+                            perbill_annual_to_perbill_round,
+                        };
+                        use $runtime::parachain_params::DefaultBlocksPerRound;
+
+                        perbill_annual_to_perbill_round(
+                            annual,
+                            // rounds per year
+                            u32::try_from(zeitgeist_primitives::constants::BLOCKS_PER_YEAR).unwrap() / DefaultBlocksPerRound::get()
+                        )
+                    }
+                    let annual = pallet_parachain_staking::inflation::Range {
+                        min: annual_inflation_min,
+                        ideal: annual_inflation_ideal,
+                        max: annual_inflation_max,
+                    };
+                    pallet_parachain_staking::inflation::InflationInfo {
+                        // staking expectations
+                        expect: pallet_parachain_staking::inflation::Range {
+                            min: Perbill::from_percent(5).mul_floor(total_supply),
+                            ideal: Perbill::from_percent(10).mul_floor(total_supply),
+                            max: Perbill::from_percent(15).mul_floor(total_supply),
```

Comment:
For the migration to Polkadot we also migrate our balances storage to the new runtime. This means we also "copy" the current `totalIssuance`.

We have a difference here for this, which is calculated with `total_supply - totalIssuance` of:

`1021933231948330118 - 1000000000000000000 = 21933231948330118` ZTG

This currently means, that there is an inflation difference. I know this can be changed by governance with [`set_inflation`](https://github.com/PureStake/moonbeam/blob/358538033db53aa8b0d38737eb560f6bb58c4390/pallets/parachain-staking/src/lib.rs#L777-L799) or [`set_staking_expectations`](https://github.com/PureStake/moonbeam/blob/358538033db53aa8b0d38737eb560f6bb58c4390/pallets/parachain-staking/src/lib.rs#L757-L776). So it should be not a big of deal, right?

Reply:
We will use the same inflation config on Polkadot as on Kusama. Those values can be used for next networks or can be adjusted if necessary.